kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/settings/ServerCertsTabUi.java

```diff
@@ -110,6 +110,11 @@ public void refresh() {
         }
     }
 
+    @Override
+    public void clear() {
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"